### PR TITLE
Fixing issue with default GCE master type when testing with kops

### PR DESCRIPTION
**Pull request by chrislovecnm**
We are setting a default master type now for AWS, and this overrides
the master type for GCE. This PR fixes that.

Fixes: https://github.com/kubernetes/test-infra/issues/7322

### CODE REVIEW

File: kubetest/kops.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ import (
 )
 
 var (
+	// kopsAWSMasterSize is the default ec2 instance type for kops on aws
+	kopsAWSMasterSize = "c4.large"
```
Comment (krzyzacy) [inline kubetest/kops.go]: nit: const?

Comment (chrislovecnm) [reply kubetest/kops.go]: Not a nit pick ... it should be :D

Comment (chrislovecnm) [reply kubetest/kops.go]: Fixed

**Comment by krzyzacy** [comment]
does this make a difference to kops-gce job? I assume that job will fall into the if statement thus ends up with same flags?

**Comment by chrislovecnm** [comment]
@krzyzacy I killed the kops gce job. It is currently busted :(

**Pull request merged**